app/src/main/java/org/thoughtcrime/securesms/conversation/disappearingmessages/ui/DisappearingMessages.kt

```diff
@@ -43,13 +45,17 @@ fun DisappearingMessages(
         Box(modifier = Modifier.weight(1f)) {
             Column(
                 modifier = Modifier
-                    .padding(bottom = 20.dp)
+                    .padding(vertical = LocalDimensions.current.spacing)
                     .verticalScroll(scrollState)
                     .fadingEdges(scrollState),
-                verticalArrangement = Arrangement.spacedBy(LocalDimensions.current.smallSpacing)
             ) {
-                state.cards.forEach {
-                    OptionsCard(it, callbacks)
+                state.cards.forEachIndexed { index, option ->
+                    OptionsCard(option, callbacks)
+
+                    // add spacing if not the last item
+                    if(index != state.cards.lastIndex){
+                        Spacer(modifier = Modifier.height(LocalDimensions.current.spacing))
```

Comment:
what's wrong with `spacedBy`?

Reply:
It spaces everything with that value but that isn't what we want here because spaceBy will also space the _**content**_ of OptionsCard since you have them openly in the ColumnScope in there.

Reply:
oh, right, my bad, should we just put some `Column` in options card then? I think I was trying to be efficient, but confusing is worse.

Reply:
spaceBy would also affect the footer which has a smaller spacing